ldk/javascript/examples/self-test-loop/src/tests/browser/index.ts

```diff
@@ -143,6 +143,34 @@ export const testOpenWindowAndListenNavigation = (): Promise<boolean> =>
     }
   });
 
+export const testTabChangeEvent = (): Promise<boolean> =>
+  new Promise(async (resolve, reject) => {
+    try {
+      const URL = 'https://www.oliveai.dev/';
+      const URL2 = 'https://www.oliveai.com/';
+      const TITLE = 'Olive Helps Developer Hub';
+
+      await browser.openTab(URL);
+      await browser.openTab(URL2);
+
+      const listener = await browser.listenTabChange((tabChangeDetails: TabChangeDetails): void => {
+        const { tabId, title, url, windowId } = tabChangeDetails;
+        if (tabId && windowId && title === TITLE && url === URL) {
+          listener.cancel();
+          resolve(true);
+        }
+
+        setTimeout(() => {
+          // After 3 seconds, if tabs havent been switched throw error
+          reject(new Error('The text selection in your browser does not match the test text'));
```

Comment:
I think this error message needs to be updated

Reply:
update message